### DISCUSSION TITLE
Upgrade Guide & V9 missing on Spatie.be

### DISCUSSION THREAD

**Discussion by JapSeyz** [spatie/laravel-backup#1827] [General]
Hey,

I am looking for an upgrade guide for V9.

The version selector on Spatie.be https://spatie.be/docs/laravel-backup/v8/introduction only goes up to v8 as of now.

**Comment by ekandreas** [comment]
Some new errors on upgrade from v8 to v9. It would be great with some instructions before going there...

**Reply by OliverGrimsley** [reply]
Yep, I am not going to risk just looking at the config file for v9 and hoping it works, as I do not know what else changed. An upgrade guide somewhere - or even just a reference that if you conform the config file from 8 to 9  will work - would be great. 

Mine failed because a number of keys in my config are not there (in my case `relative_path` was not in my config), but in looking at my v8 config to v9, there are many other keys in the v9 code that seem to be required now.

**Reply by snapey** [reply]
One year on, still no upgrade instructions...
